.github/workflows/comment.yml

```diff
@@ -0,0 +1,15 @@
+on: pull_request
+
+jobs:
+  example_comment_pr:
+    runs-on: ubuntu-latest
+    name: An example job to comment a PR
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Comment PR
+        uses: thollander/actions-comment-pull-request@v2
+        with:
+          message: |
+            Thank you for your pull request! As this is a sandbox repository, your pull request will eventually be closed. If you would like engagement from the Pythia team on this, please tag `@ProjectPythia/infrastructure` to let us know!
```

Comment:
I don't think people from outside the org can tag teams. conda-forge has some bots around to enable that very use case. Not sure that we want to go that far, but probably don't want the message to suggest something that can't be done.

Reply:
Oh yeah, forgot about that!

Maybe we should just remove all mention of tags here. The welcome message can just say clearly that users should *not* expect any response in the sandbox. We don't really have resources to provide open-ended support to people learning git / github anyway.

Reply:
Took that out from both actions. Good catch!